recipe/build.sh

```diff
@@ -1,23 +1,8 @@
 #!/usr/bin/env bash
-
-cd "${SRC_DIR}/.."
-mkdir "src"
-mkdir "src/github.com"
-mkdir "src/github.com/opencontainers"
-mkdir "src/github.com/tianon"
-mv "${SRC_DIR}" "src/github.com/tianon/gosu"
-
-export RUNC_VERSION="0.1.0"
-curl -L https://github.com/opencontainers/runc/archive/v${RUNC_VERSION}.tar.gz | tar -xzC "src/github.com/opencontainers"
-mv "src/github.com/opencontainers/runc-${RUNC_VERSION}" "src/github.com/opencontainers/runc"
-
 export CGO_ENABLED=0
 export GOARCH="amd64"
 export GOOS="linux"
-export GOPATH="$(pwd)"
+export GOPATH=$SRC_DIR
 
-cd "src/github.com/tianon/gosu"
+pushd "$GOPATH/src/github.com/tianon/gosu"
 go build -v -ldflags '-d -s -w' -o "${PREFIX}/bin/gosu"
-
-mkdir -p "${SRC_DIR}"
-cp "LICENSE" "${SRC_DIR}/LICENSE"
```

Comment:
License code can be pushed to meta.yaml, thus reducing complexity of build.sh as well.

Reply:
Yeah, `conda-build` has gotten a lot smarter since this was written. 😉